src/redux/books/bookapi.js

```diff
@@ -0,0 +1,68 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+// gOZ6ayJ9IcL8x7y2ImTP
```

Comment:
- [ Required ] Please consider removing any commented code from your codebase, this makes your codebase look unprofessional. Please rectify this.

Reply:
Fixed.